WEB_SERVICE/WEB/_HIGH/GRAPHQL_DIRECTIVE_OVERLOADING/recommendation.md

```diff
@@ -0,0 +1,58 @@
+To mitigate the risk of Directive Overloading attacks, you can take the following steps:
+
+- **Rate Limiting**: Apply rate-limiting to prevent a large volume of requests from overwhelming your server with excessive directive usage.
```

Comment:
it doesn't help in this case because , using just one request with soooo many @ can dos the server

Reply:
make sense, addressed.